projects/test-app/src/app/app.component.ts

```diff
@@ -11,5 +11,6 @@ export class AppComponent {
 
   constructor(public customers: CustomerFacade) {
     customers.loadAll();
+    setTimeout(() => customers.clear(), 3000);
```

Comment:
Should this line still be here or was it purely for testing?

Reply:
This is jus a test app. It is simpler than the demo app, and I've been using it just to play around with things in a real-world context. The timeout here was so that the clear action was dispatched after the load action was handled. Without the timeout, clear was being sent after Load but before LoadSuccess was dispatched.

Since it is just a test app, this code doesn't really matter. It'll also probably change in the future for testing different features. I also encourage you guys to use it, if you need to play with a feature! ;)

Reply:
Sorry, hadn't noticed it wasn't the demo app. Woops!